app/controllers/application_controller.rb

```diff
@@ -296,7 +296,7 @@ def saved_redirect_url_or(default)
 
   def generic_exception(exception)
     logger.warn "Operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.error exception.backtrace.join("\n")
```

Comment:
A few more thoughts...

1. If it's at the same log level, we can log it as a single multi-line message so they don't become separated.
2. We could run this backtrace through the silencer, making it shorter (but thanks to your other changes, still accurate).  If debug's enabled, then it could be non-silenced and longer?
3. Combine this into one easy to use helper :)